docs/dev/build-instructions/faq.md

```diff
@@ -20,3 +20,35 @@ If for instance you have installed OpenSSL to `/usr/local/opt/openssl`, you can
 cmake -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl ..
 ```
 
+
+### Build a specific version or commit
+
+#### Version
+
+Use tags to build a specific release version of Mumble.
+You can find the `tag`-list [here](https://github.com/mumble-voip/mumble/tags) or by using the command `git tag` inside a cloned directory.
+
+Use:
+
+`git clone -b [tag] https://github.com/mumble-voip/mumble.git`
+
+For example: 
+`git clone -b 1.3.3 https://github.com/mumble-voip/mumble.git` (for version 1.3.3)
+
+**or inside a cloned directory:**
+
+`git checkout [tag]` 
+
+or `git checkout -b [new-branch-name] [tag]` (if you want to create a new branch)
+
+#### Commit
+
+Use commit hashes to build a specific development state of Mumble.
+
+You can find the list of commits [here (for master branch)](https://github.com/mumble-voip/mumble/commits/master) or by using the command `git log` inside a cloned directory.
```

Comment:
```suggestion
You can find the list of commits [here (for master branch)](https://github.com/mumble-voip/mumble/commits/master) or by using the command `git log` inside a cloned repository.
```

Reply:
Thats funny, because I chose the name "repo" first, but then I saw that you use "cloned directory" everywhere, so I changed it.

Reply:
I did? :eyes: 

Ah yes in the main file I did indeed. I think that maybe the context is a little bit different there. However looking at it now I think I should have written `repository` there as well :thinking:

Reply:
I can change it, if you agree.